custom_components/tuya_local/devices/smart_weather_station.yaml

```diff
@@ -0,0 +1,93 @@
+name: Smart Weather Station
+products:
+  - id: 000001bbfd
```

Comment:
Is this the product id?  It looks more like what appears in the modelId field of Query Things Data Model API call, which is something different (maybe some internal id for Tuya cloud).

Reply:
it is going to be changed to "fpgistthfiweyaqr"